virttest/libvirt_xml/nodedev_xml.py

```diff
@@ -140,8 +140,27 @@ class for capability whose type is pci.
     #  <numa node='0'/>
     #</capability>
 
+    # or the Example of iommu:
+    #<capability type="pci">
```

Comment:
follow the comment format(leave one white space between # and following words):# xx ,and the following have similar issues.

Reply:
Updated